optuna/pruners/hyperband.py

```diff
@@ -124,7 +124,7 @@ def __init__(
         _logger.debug("Hyperband has {} brackets".format(self._n_brackets))
 
         for i in range(self._n_brackets):
-            bracket_resource_budget = self._calc_bracket_resource_budget(i, self._n_brackets)
+            bracket_resource_budget = self._calc_trial_allocation_budget(i)
```

Comment:
Great, it makes sense trying to avoid "resource" here. I have some suggestions. 
- How about also renaming associated variables? I.e. something along the following.
```python
            trial_allocation_budget = self._calc_trial_allocation_budget(i).  # or just `budget`.
            self._total_trial_allocation_budget += bracket_resource_budget
            self._trial_allocation_budgets.append(trial_allocation_budget)
```
-  We can spell out "calc" as "calculate".
```python
            trial_allocation_budget = self._calculate_trial_allocation_budget(i)
```